singlePost.php

```diff
@@ -39,5 +42,55 @@
         </div>
     </article>
 </section>
+
+<?php
+if(isset($_SESSION['userid']))
+{
+    ?>
+    <section class="container md:w-1/2 mx-auto mt-5">
+        <form class="p-8 border border-solid rounded-md bg-slate-200" method="post">
+            <div class="mb-5">
+                <label class="mb-3 block" for="content">Comment:</label>
+                <textarea class="w-full" id="content" name="content" rows="5"></textarea>
+            </div>
+            <?php
+            if(isset($_POST['content']))
+            {
+                // inputed special characters won't be interpreted as HTML tags/entities.
+                $content = htmlspecialchars($_POST['content'], ENT_QUOTES, 'UTF-8');
```

Comment:
As an extra safety, can you run content through strip_tags() as well - it'll remove any HTML tags in the content

Reply:
done